addons/web_editor/static/src/xml/wysiwyg.xml

```diff
@@ -66,7 +66,7 @@
                                 </div>
                             </div>
                             <div class="mb-3 row">
-                                <label class="col-form-label col-md-3">Style</label>
+                                <label class="col-form-label col-md-3">Layout</label>
```

Comment:
This has nothing to do with the shortcuts and should be in another commit. I suggest to put it in the commit where Type is removed to Style since it is the reason why this other change had to be made as well.